server/src/main/java/com/networknt/server/Server.java

```diff
@@ -256,35 +263,36 @@ static private boolean bind(HttpHandler handler, int port) {
                 SwitcherUtil.setSwitcherValue(Constants.REGISTRY_HEARTBEAT_SWITCHER, true);
                 if (logger.isInfoEnabled())
                     logger.info("Registry heart beat switcher is on");
-
-            }
-
-            if (config.enableHttp) {
-                System.out.println("Http Server started on ip:" + config.getIp() + " Port:" + port);
-                if (logger.isInfoEnabled())
-                    logger.info("Http Server started on ip:" + config.getIp() + " Port:" + port);
-            } else {
-                System.out.println("Http port disabled.");
-                if (logger.isInfoEnabled())
-                    logger.info("Http port disabled.");
-            }
-            if (config.enableHttps) {
-                System.out.println("Https Server started on ip:" + config.getIp() + " Port:" + port);
-                if (logger.isInfoEnabled())
-                    logger.info("Https Server started on ip:" + config.getIp() + " Port:" + port);
-            } else {
-                System.out.println("Https port disabled.");
+                // handle the registration exception separately to eliminate confusion
+            } catch (Exception e) {
+                System.out.println("Failed to register on Consul, server stopped.");
                 if (logger.isInfoEnabled())
-                    logger.info("Https port disabled.");
+                    logger.info("Failed to register on Consul, server stopped.");
```

Comment:
There shouldn't be any reference to Consul in here. The server isn't locked into a specific registration service